releases/release-1.25/release-notes/maps/pr-111479-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 111479
+releasenote:
+  text: CSIMigrationAWS upgraded to GA and locked to true.
```

Comment:
```suggestion
  text: `CSIMigrationAWS` upgraded to GA and locked to true.
```

Reply:
We can't use the backtick `  as the first character in a scalar, @ and ` are reserved characters https://yaml.org/spec/1.2.2/#rule-c-reserved

@Divya063 recently hit this issue here https://github.com/kubernetes/sig-release/pull/1975/files